app/components/channel_link/channel_link.js

```diff
@@ -74,7 +74,7 @@ export default class ChannelLink extends React.PureComponent {
                     style={this.props.linkStyle}
                     onPress={this.handlePress}
                 >
-                    {channel.display_name}
+                    {'~' + channel.display_name}
```

Comment:
According to mattermost style guide, template strings are preferred to concating.
https://docs.mattermost.com/developer/style-guide.html

So, it should be 

    `~${channel.display_name}`

Reply:
Not sure it is block-worthy though. I am only 1/5

Reply:
agree with @stephenkiers